facts/aps/aplist.csv

```diff
@@ -61,7 +61,7 @@ pacific-2,n8t-0072,c4:04:15:9b:5e:b4,10.0.3.76,6,48,0,0,50,50
 pacific-3,n8t-0027,c4:04:15:8f:30:72,10.0.3.111,1,44,0,0,50,50
 pacific-4,n8t-0105,08:bd:43:ac:5f:6c,10.0.3.64,1,36,0,0,50,50
 pacific-5,n8t-0067,c4:04:15:9b:4c:03,10.0.3.123,11,48,0,0,50,50
-plaza-1,n8c-0007,04:a1:51:8b:0f:f9,10.0.3.44,6,161,0,0,50,50
+plaza-1,n8t-0055,08:bd:43:b0:ea:7b,10.0.3.44,6,161,0,0,50,50
```

Comment:
n8c-0007 should be marked as spare not removed

Reply:
I cannot find any other record of 0007, I'm not sure it actually exists.

Reply:
talked with David, we will keep `n8c-0007` since its still present in inventory